lib/reply.js

```diff
@@ -124,7 +124,7 @@ Reply.prototype.send = function (payload) {
   }
 
   if (this.sent) {
-    this.log.warn({ err: new FST_ERR_REP_ALREADY_SENT() }, 'Reply already sent')
+    this.log.warn({ err: new FST_ERR_REP_ALREADY_SENT(this.request.url) })
```

Comment:
Should we include the `method` here to distinguish between handlers?
Asking as is possible the same route is bounded to different methods